addons/web/static/src/utils/dates.js

```diff
@@ -144,12 +251,12 @@ const smartDateRegex = new RegExp(`^([+-])(\\d+)([${Object.keys(dateUnits).join(
  *   "-4y" will return now + 4 years
  *
  * @param {string} value
- * @returns {DateTime|false} Luxon datetime object
+ * @returns {DateTime|false} Luxon datetime object (in the UTC timezone)
  */
 export function parseSmartDateInput(value) {
   const match = smartDateRegex.exec(value);
   if (match) {
-    let date = DateTime.local();
+    let date = DateTime.utc();
```

Comment:
are you sure of this?

Reply:
Yes, the only place `parseSmartDateInput` is used is in the `parseDate`/`parseDateTime` methods (also true for legacy).